src/include/color.h

```diff
@@ -4,10 +4,11 @@
 struct ColorLayout
 {
     /* The order is actually important - if it's identical to memory layout of our render surface, it's just a memcopy*/
+    /* and it will be UB, if you do memcopy :) */
```

Comment:
Je to tak, ale optimalizator MSVC sucks na tomhle a vygeneroval mi tam odporne pomaly loop. Clang v tomhle lepsi. Zatim pro MSVC memcopy, navrhni mi stejne rychle, ne-UB reseni :)